Lib/asyncio/tasks.py

```diff
@@ -104,6 +120,11 @@ def __init__(self, coro, *, loop=None):
             self._log_destroy_pending = False
             raise TypeError(f"a coroutine was expected, got {coro!r}")
 
+        if name is None:
+            self._name = 'Task-%s' % _task_name_counter()
```

Comment:
nit: I've recently updated asyncio to use f-strings. Please change this one to either an f-string or to a `.format()` call (just for maintaining consistency of the codebase).